### PR TITLE
Additional dependencies

### PR DESCRIPTION
**Pull request by Silas-Ricky**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -49,8 +49,8 @@
 
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
-			<artifactId>inventory-api</artifactId>
-			<version>2.0.10-SNAPSHOT</version>
```
Comment (ningosi) [inline api/pom.xml]: This dependancies need to get to the main pom.xml

File: api/target/maven-java-formatter-cache.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Tue Jul 14 16:07:34 EAT 2020
+#Tue Jul 14 19:51:21 EAT 2020
 /src/main/java/org/openmrs/module/ehrcashier/billcalculator/common/BillCalculatorImpl.java=76f3247421305fc589930c82466bd26f
```
Comment (ningosi) [inline api/target/maven-java-formatter-cache.properties:2]: We don't need this file, please clean up

File: pom.xml [hunk 3]
```diff
@@ -42,7 +42,7 @@
 		<uiframeworkVersion>3.3.1</uiframeworkVersion>
 		<kenyacoreVersion>2.4.1</kenyacoreVersion>
 		<kenyauiVersion>2.1.2-SNAPSHOT</kenyauiVersion>
-		<kenyaemrVersion>17.0.3</kenyaemrVersion>
+		<kenyaemrVersion>17.1.0</kenyaemrVersion>
```
[the remark this answers is not in the document]
Comment (Silas-Ricky) [reply pom.xml:47]: I have changed it to kenyaEMR Version 17.0.3

File: omod/src/main/java/org/openmrs/module/ehrcashier/page/controller/PatientServiceVoidedBillViewForBDPageController.java [hunk 4]
```diff
@@ -21,80 +21,75 @@
...
+	        UiUtils uiUtils) {
+		BillAccess ba = new BillAccess();
+		boolean auth = ba.authenticate(pageRequest, sessionContext);
+		if (!auth) {
+			return "redirect: index.htm";
+		}
+		BillingService billingService = Context.getService(BillingService.class);
+		Patient patient = Context.getPatientService().getPatient(patientId);
+		Map<String, String> attributes = PatientUtils.getAttributes(patient);
+		//ghanshyam 25-02-2013 New Requirement #966[Billing]Add Paid Bill/Add Free Bill for Bangladesh module
```
[the remark this answers is not in the document]
Comment (Silas-Ricky) [reply omod/src/main/java/org/openmrs/module/ehrcashier/page/controller/PatientServiceVoidedBillViewForBDPageController.java:45]: Removed

File: pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Silas-Ricky) [reply pom.xml]: Sorry I never mentioned earlier that version 17.1.0 is the latest version currently being used

File: api/src/main/java/org/openmrs/module/ehrcashier/billcalculator/BillCalculatorForBDService.java [hunk 5]
```diff
@@ -17,9 +17,9 @@
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.openmrs.module.hospitalcore.util.GlobalPropertyUtil;
-import org.openmrs.module.hospitalcore.util.HospitalCoreConstants;
-
+/*import org.openmrs.module.hospitalcore.util.GlobalPropertyUtil;
+*import org.openmrs.module.hospitalcore.util.HospitalCoreConstants;
```
[the remark this answers is not in the document]
Comment (Silas-Ricky) [reply api/src/main/java/org/openmrs/module/ehrcashier/billcalculator/BillCalculatorForBDService.java]: Corrected

File: api/pom.xml (hunk 1 above)
Comment (Silas-Ricky) [reply api/pom.xml]: Resolved

File: omod/pom.xml [hunk 6]
```diff
@@ -34,7 +34,18 @@
...
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
-			<artifactId>referenceapplication-api</artifactId>
+			<artifactId>inventoryapp-api</artifactId>
+			<version>1.0.2-SNAPSHOT</version>
+			<type>jar</type>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
```
[the remark this answers is not in the document]
Comment (Silas-Ricky) [reply omod/pom.xml:51]: Ported files from reference application module successfully

**Pull request merged**